wpeverest/everest-forms
# Enhancement - Move everest form pro fields into free

### All Submissions:


### Changes proposed in this Pull Request:

This PR moves the some field of the everest form pro into free. The moved field list is listed on the jira card.
https://themegrill.atlassian.net/browse/EVF-965

### Dependancy PR:
https://github.com/wpeverest/everest-forms-pro/pull/887

### How to test the changes in this Pull Request:

1. Switch into the branch and deactivate the everest form pro and survey addon.
2. Verify the listed field are able to use in the everest form free.
3. Verify the field by changing the setting of each field and also check the backward compatibility.

### Types of changes:


### Other information:


### Changelog entry

> Enhancement - Move everest form pro fields into free.
